config/config.go

```diff
@@ -50,12 +50,23 @@ type Company struct {
 	People  []Account `json:"people" yaml:"people"`
 }
 
-// ClaSigners provides the overall structure of the CLA config: individual CLA
-// signers, bots, and corporate CLA signers.
-type ClaSigners struct {
+// ExternalClaSigners represents CLA signers managed by an external process,
+// i.e., not covered by this tool. This is useful for handling migrations into
+// or out of the system provided by Code Review Bot.
+type ExternalClaSigners struct {
 	People    []Account `json:"people,omitempty" yaml:"people,omitempty"`
 	Bots      []Account `json:"bots,omitempty" yaml:"bots,omitempty"`
 	Companies []Company `json:"companies,omitempty" yaml:"companies,omitempty"`
+	AllOthers bool      `json:"all_others" yaml:"all_others"`
```

Comment:
The config JSON/YAML is for things like secrets as well as org/repo config, and thus contains things different from the content of the CLA config, whereas this boolean affects everything else that is defined in the same CLA YAML file, so I think it should be present there.

In practice, for JanusGraph, we version and keep the CLA file publicly, while the config file is kept privately (since it includes secrets such as the API key), so it seems strange to hide a parameter that significantly affects the behavior of the tool, but isn't needed to be kept secret — it should be part of the public record and code reviewed as the rest of the behavior-changing configs, such as adding/removing users from the CLA list.

Reply:
Having thought about this some more, maybe it makes sense to factor out the secret part of the config into a separate file (and a different flag), say `-secrets` or `-auth`, such that config file can be made public and then it can include such a boolean flag there. What do you think?

Reply:
Filed https://github.com/google/code-review-bot/issues/47 to track this; will implement that first and then come back here and refactor this PR to use the global config setting instead of embedding the `all_others` field in the CLA signers file.

Reply:
Implemented #47 and updated this PR to use the new functionality from config instead of bundling it in the CLA signers file. Thanks again for the suggestion!